libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/AppLockView/UIColor+SFSDKPasscodeView.m

```diff
@@ -30,15 +30,47 @@
 @implementation UIColor (SFSDKPasscodeView)
 
 + (UIColor *)passcodeViewBackgroundColor {
-    return [UIColor colorWithRed:245.0f/255.0f green:246.0f/255.0f blue:250.0f/255.0f alpha:1.0f];
+    UIColor *lightStyleColor = [UIColor colorWithRed:245.0f/255.0f green:246.0f/255.0f blue:250.0f/255.0f alpha:1.0f];
```

Comment:
Not for this PR, but we should consider combining all the UIColor+ classes or deriving colors from a single source of truth.

Reply:
Sounds good! I can do it as a part of the IDP changes because it’ll need the light/dark helper method anyway